lib/fog/openstack/requests/orchestration/validate_template.rb

```diff
@@ -11,6 +11,14 @@ def validate_template(options = {})
           )
         end
       end
+      class Mock
+        def validate_template(options = {})
+          Excon::Response.new(
+            :body   => { 'description' => "Valid Stack"},
```

Comment:
response here should have 

{"Description" => ..., "Parameters" => ... , "ParameterGroups" => ...}